themes/bootstrap3/templates/combined/results-list.phtml

```diff
@@ -15,6 +15,18 @@
     ? $this->url($advancedSearchAction, [], ['query' => ['edit' => $results->getSearchId()]])
     : false;
 ?>
+<?php
+    $classId = $domId ?? 'combined_' . $searchClassId;
+    $countsJs = <<<JS
+        $(document).ready(function() {
```

Comment:
Would it be easy to do this without jQuery? Since we're trying to move away from jQuery, it might be better to avoid it here so we don't need to port code later... but if it's a lot of work to convert now, I don't insist on the change. :-)

Reply:
Ok, I rewrote the code to avoid jQuery.  I used several design patterns that are possibly better to extract into a separate JS module, but that would introduce race conditions and other complexity, so not doing that for now.

Used as guidance:

- https://youmightnotneedjquery.com/#ready
- https://youmightnotneedjquery.com/#create_elements

There is another part of the ready() logic needed to support IE <= 8, but I'm glad to see we don't support that.